### DISCUSSION TITLE
为什么生成的中文含有很多乱码？

### DISCUSSION THREAD

**Discussion by linonetwo** [Atome-FE/llama-node#100] [Q&A]
```wikitext

* TiddlyWiki是一种基于HTML的���������记本，���可以在������器中运行。
* ���的特点是：
	+ ���活性���：用户可以自由编���和���展TiddlyWiki的内容。
	+ ���用性���：TiddlyWiki的用户界面���单������，不需要长时间学���。
	+ 可���展性���：用户可以添加自���的���件和���展功能。
	+ 可���性���：TiddlyWiki使用文件和本地数据库存���数据，可以在网���不���定的������下������使用。
```

我尝试了 ggml-vic7b-q5_1.bin ~~和 RWKV-4-World-CHNtuned-7B-v1-20230709-ctx4096-Q5_0.bin ，都是这样的效果。~~ rwkv 其实失败了 #101

**Comment by Zacharia2** [comment]
` ./main -m ../ggml-vic7b-q5_1.bin --color -ins -p "You known much on TiddlyWiki. You should answer in wikitext format(`!` for title, and `#` for numeral list, `*` for unordered list.)." --temp 1`

![image](https://github.com/Atome-FE/llama-node/assets/32425955/17677d1c-e7a0-43f0-a997-1f2468d3f2f6)

在控制台好像没有乱码。

**Reply by linonetwo** [reply]
你这是用 llama-node 运行的吗？那看来是太记app在前后端传输的时候出了问题

**Reply by Zacharia2** [reply]
https://github.com/ggerganov/llama.cpp

这个

**Reply by linonetwo** [reply]
<img width="1203" alt="截屏2023-07-19 13 53 58" src="https://github.com/Atome-FE/llama-node/assets/3746270/f8952c1e-e08b-486d-965e-03e2875f4924">

我用 llama-node 网站上的代码就不行 https://llama-node.vercel.app/docs/backends/llama.cpp/inference

看来是 llama-node 的问题

**Reply by linonetwo** [reply]
@Zacharia2 为什么我运行 `./chat_mac --model "/Users/linonetwo/Documents/languageModel/ggml-vic7b-q5_1.bin"  --threads 4  --prompt "介绍一下Tiddlywiki"`

它是说 bad magic  failed to load model ，你下载模型的链接就是太记官网上给的那个跟我一样的吗


```
main: seed = 1689748090
llama_model_load: loading model from '/Users/linonetwo/Documents/languageModel/ggml-vic7b-q5_1.bin' - please wait ...
llama_model_load: invalid model file '/Users/linonetwo/Documents/languageModel/ggml-vic7b-q5_1.bin' (bad magic)
main: failed to load model from '/Users/linonetwo/Documents/languageModel/ggml-vic7b-q5_1.bin'
```

**Reply by Zacharia2** [reply]
对，就是这个名字的模型

**Reply by Zacharia2** [reply]
我是直接下载的，Mac可能需要自己编译

**Reply by Zacharia2** [reply]
推理程序
